sdk/textanalytics/Azure.AI.TextAnalytics/src/Transforms.cs

```diff
@@ -329,7 +329,9 @@ internal static PiiTask ConvertToPiiTask(RecognizePiiEntitiesOptions option)
                 {
                     Domain = option.DomainFilter.HasValue ? option.DomainFilter.Value.GetString() : (PiiTaskParametersDomain?)null,
                     ModelVersion = !string.IsNullOrEmpty(option.ModelVersion) ? option.ModelVersion : "latest",
-                    StringIndexType = option.StringIndexType
+                    StringIndexType = option.StringIndexType,
+                    LoggingOptOut = option.DisableServiceLogs.HasValue ? option.DisableServiceLogs : null
```

Comment:
I could be missing something, but can't we do `LoggingOptOut = option.DisableServiceLogs` directly?